### ISSUE TITLE
Possible duplication with info-frontend

### ISSUE THREAD

**Issue by nickcolley** [alphagov/content-performance-manager#17]
As part of discovery the Template Consolidation team has been looking at our frontend applications and seeing where we could reduce duplication. (https://gov-uk.atlassian.net/wiki/display/FE/Applications)

Looking at [info-frontend](https://github.com/alphagov/info-frontend) we've found there is a small amount of use (~300 visits per month), it would seem that the needs that it is serving could overlap with the content-performance-manager.

Has your team done any research around [info-frontend](https://github.com/alphagov/info-frontend)'s users? There may be a potential to remove it if the needs can be met by this applications.

**Comment by selfthinker** [comment]
We discussed this last week within the team. @pmanrubia looked at info-frontend (and other repos) quite closely. He found that it overlaps to an extend but not in a way which is very useful for us.

This app looks at an overview of departments' content first, later drilling down to single items. info-frontend is only looking at single items. Although we are interested at some point in the 5 metrics info-frontend produces, for this app we are interested in much, much more.

Thanks to info-frontend we decided to use the same API it's using (metadata-api) and potentially enhance it later or switch to something else or a mixture of the two.

It might happen that this app will link to info-frontend and connect to it in that way in the future. But it's more likely this app will provide the same data as info-frontend for single pages later. Then info-frontend could indeed just be deleted.
